x-pack/plugins/threat_intelligence/cypress/integration/indicators/indicators.spec.ts

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
this is just moving `indicators.spec.ts` from `sources` to `indicators` directory, as `sources` is just an old term used for feeds e2e test initially